git/gitgithubclient.py

```diff
@@ -40,7 +39,7 @@ def _commitData(self, commit):
 			"message": commit.commit.message
 		}
 
-	def commits(self, branch, since = 0, to = datetime.datetime.now() + datetime.timedelta(hours=24)):
+	def commits(self, branch, since=0, to=0):
```

Comment:
Still, this solutions is "kind of" ugly to add new functions/methods just to handle date formats.